terrautils/metadata/lemnatec.py

```diff
@@ -1,24 +1,32 @@
-'''
-Create a cleaned/curated metadata object based on the LemnaTec fixed metadata
-Some metadata used for search
-Some metadata used for extractors
+"""
+Given a JSON object containing the original LemnaTec metadata, convert
+field names and values to standardized formats required for TERRA-REF.
```

Comment:
OK. I wasn't sure how far you had gotten with this, and am thinking at a different scale.

- my primary concern was that the current field name combines lots of information that could be parsed into fields like 'object', 'measurement', 'unit', and 'value'
- SensorML provides frameworks for defining a local coordinate system and a location, position, orientation within the coordinate system. so ... this schema from the [SensorML specification](http://portal.opengeospatial.org/files/?artifact_id=21273) seems to provide a framework for combining many of the fields listed here into a coherent model.

![image](https://user-images.githubusercontent.com/464871/29228387-1fa70b4a-7e9f-11e7-8009-9cb0c9e8c200.png)

This isn't essential to the scientific use case, and I would prioritize providing trait data in BRAPI format more important than providing sensor metadata in SensorML format. We can discuss and get feedback on the value of adopting this standard prior to the 2018 release.